app/controllers/openid_connect/discovery_controller.rb

```diff
@@ -0,0 +1,51 @@
+module OpenidConnect
+  class DiscoveryController < ApplicationController
+    def show
+      case params[:id]
+        when 'webfinger'
+          webfinger_discovery
+        when 'openid-configuration'
+          openid_configuration
+        else
+          raise HttpError::NotFound
+      end
+    end
+
+    private
+
+    def webfinger_discovery
+      jrd = {
+        links: [{
+                  rel: OpenIDConnect::Discovery::Provider::Issuer::REL_VALUE,
+                  href: absoluteURL("/")
+                }]
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.